projects/ion/src/lib/tooltip/tooltip.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { CommonModule } from '@angular/common';
+import { render, screen } from '@testing-library/angular';
+import { TooltipPosition } from '../core/types';
+import { TooltipComponent, TooltipProps } from './tooltip.component';
+
+const positions = Object.values(TooltipPosition) as TooltipPosition[];
+
+const defaultProps: TooltipProps = {
+  ionTooltipTitle: 'Title',
+};
+
+const sut = async (
+  props: Partial<TooltipComponent> = defaultProps
+): Promise<void> => {
+  await render(TooltipComponent, {
+    componentProperties: {
+      ...props,
+    },
+    imports: [CommonModule],
+  });
+};
+
+describe('TooltipComponent', () => {
+  it('should render title', async () => {
+    const ionTooltipTitle = 'Eu sou um tooltip.';
+    await sut({ ionTooltipTitle });
+    expect(screen.getByTestId('ion-tooltip')).toHaveTextContent(
```

Comment:
This was repeatable in other cases, I believe that you can create a const to this id.